packages/frontend/src/pages/ManageVolunteersPage/ManageVolunteersPage.tsx

```diff
@@ -99,53 +130,180 @@ export const ManageVolunteersPage = () => {
   return (
     isEditorHasPermissions && (
       <>
-        <div className={css.volunteersBlockWrapper}>
-          <p className={css.volunteersTitle}>{translation.volunteers}</p>
-          <div className={css.searchbarWrapper}>
-            <img className={css.searchIcon} src={MagnifyerSvg} />
+        {toggleFilterMenu && (
+          <div className={css.filterMenu}>
+            <div className={css.titleAndFilter}>
+              <p className={css.volunteersTitle}>{translation.filter}</p>
+              <button onClick={resetAllFilters} className={css.resetButton}>
+                {translation.reset}
+              </button>
+            </div>
+            <div className={css.languagesSection}>
+              <p className={css.filterSectionTitle}>{translation.languages}</p>
+              <div className={css.languageItem}>
+                <input
```

Comment:
input and labels should be connected, so when user click on label, input should be changed
`<input type="checkbox" id="english" ... /> <label for="english">English</label>`

and instead of adding {' '} gaps are usually added via css. In css class languageItem it's set to `dispay: flex`, so you can just add something like `gap: 4px;`